### ISSUE TITLE
SmartCity-Accessibility #128 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#108]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 115 lines...]
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.021 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.socialnetwork.ReviewCommentTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.036 sec - in smartcity.accessibility.socialnetwork.ReviewCommentTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.054 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.BrokenAdminTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.214 sec - in smartcity.accessibility.socialnetwork.BrokenAdminTest
Running smartcity.accessibility.socialnetwork.MockAdminTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.684 sec - in smartcity.accessibility.socialnetwork.MockAdminTest
Running smartcity.accessibility.search.SearchTest
-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: Dec 27, 2016
JxMaps license valid.
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 305.451 sec - in smartcity.accessibility.search.SearchTest
Running smartcity.accessibility.navigation.NavigationTest
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 36700160 bytes for committing reserved memory.
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00000000fdd00000, 36700160, 0) failed; error='Cannot allocate memory' (errno=12)
# An error report file with more information is saved as:
# /var/lib/jenkins/workspace/SmartCity-Accessibility/hs_err_pid11101.log

Results :

Tests run: 28, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 05:51 min
[INFO] Finished at: 2016-12-27T14:38:09+00:00
[INFO] Final Memory: 28M/202M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /var/lib/jenkins/workspace/SmartCity-Accessibility && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec -jar /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefirebooter1581988199871465811.jar /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefire4825703352469218394tmp /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefire_0745688569574621517tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/128/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 115 lines...]
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.socialnetwork.ReviewCommentTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.017 sec - in smartcity.accessibility.socialnetwork.ReviewCommentTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.037 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.BrokenAdminTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.191 sec - in smartcity.accessibility.socialnetwork.BrokenAdminTest
Running smartcity.accessibility.socialnetwork.MockAdminTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.539 sec - in smartcity.accessibility.socialnetwork.MockAdminTest
Running smartcity.accessibility.search.SearchTest
-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: Dec 27, 2016
JxMaps license valid.
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 305.139 sec - in smartcity.accessibility.search.SearchTest
Running smartcity.accessibility.navigation.NavigationTest
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00000000fc680000, 38797312, 0) failed; error='Cannot allocate memory' (errno=12)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 38797312 bytes for committing reserved memory.
# An error report file with more information is saved as:
# /var/lib/jenkins/workspace/SmartCity-Accessibility/hs_err_pid11426.log

Results :

Tests run: 28, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 05:47 min
[INFO] Finished at: 2016-12-27T14:44:08+00:00
[INFO] Final Memory: 28M/179M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /var/lib/jenkins/workspace/SmartCity-Accessibility && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec -jar /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefirebooter1075189967187034154.jar /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefire5162494766455912505tmp /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefire_07375455909269368687tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Ariel Kolikant] 329231162d2284e45d8940367fdab8f4398e3882 - refactoring



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/129/)

**Comment by kapalex** [comment]
Ignore the current errors, testing something with memory management

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 115 lines...]
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.socialnetwork.ReviewCommentTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 sec - in smartcity.accessibility.socialnetwork.ReviewCommentTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.034 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.BrokenAdminTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.201 sec - in smartcity.accessibility.socialnetwork.BrokenAdminTest
Running smartcity.accessibility.socialnetwork.MockAdminTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.895 sec - in smartcity.accessibility.socialnetwork.MockAdminTest
Running smartcity.accessibility.search.SearchTest
-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: Dec 27, 2016
JxMaps license valid.
Tests run: 2, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 3.037 sec - in smartcity.accessibility.search.SearchTest
Running smartcity.accessibility.navigation.NavigationTest
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00000001003e0000, 131072, 0) failed; error='Cannot allocate memory' (errno=12)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 131072 bytes for committing reserved memory.
# An error report file with more information is saved as:
# /var/lib/jenkins/workspace/SmartCity-Accessibility/hs_err_pid12011.log

Results :

Tests run: 29, Failures: 0, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 45.312 s
[INFO] Finished at: 2016-12-27T15:02:51+00:00
[INFO] Final Memory: 28M/204M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /var/lib/jenkins/workspace/SmartCity-Accessibility && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec -jar /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefirebooter8739866322391592854.jar /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefire1952670843195011951tmp /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire/surefire_05463527085957346244tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Ariel Kolikant] 329231162d2284e45d8940367fdab8f4398e3882 - refactoring

- [Ariel Kolikant] 051a2792321fa3272159a5c851226d8af54e7a31 - added dispose 394



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/131/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
